invenio_rdm_records/config.py

```diff
@@ -63,16 +63,23 @@ def always_valid(identifier):
         "validator": idutils.is_ads,
         "datacite": "bibcode",
     },
+    "crossreffunderid": {
```

Comment:
ping @slint do we want to merge these variables?

Reply:
I think it's fine, references are kind of like `related_identifiers` so they should both accept the same types of identifiers.